docs/tutorials/get_started.ipynb

```diff
@@ -0,0 +1,240 @@
+      ],
+      "text/plain": [
+       "<Figure size 454.517x284.278 with 1 Axes>"
+      ]
+     },
+     "execution_count": 1,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "from qiskit import QuantumCircuit\n",
+    "\n",
+    "# Create a new circuit with 2 qubits and 2 classical bits (cbits)\n",
+    "qc = QuantumCircuit(2, 2)\n",
+    "\n",
+    "# Add a Hadamard gate to qubit 0\n",
+    "qc.h(0)\n",
+    "\n",
+    "# Perform a controlled-X gate on qubit 1, controlled by qubit 0\n",
+    "qc.cx(0, 1)\n",
+    "\n",
+    "# Measure qubit 0 to cbit 0, and qubit 1 to cbit 1\n",
+    "qc.measure(0, 0)\n",
+    "qc.measure(1, 1)\n",
+    "\n",
+    "# Return a drawing of the circuit using MatPlotLib (\"mpl\").\n",
+    "# Remove the \"mpl\" argument to get a text drawing.\n",
+    "qc.draw(\"mpl\")"
+   ]
+  },
+  {
+   "cell_type": "raw",
+   "id": "0c957de9",
+   "metadata": {
+    "raw_mimetype": "text/restructuredtext"
+   },
+   "source": [
+    "See :class:`~.QuantumCircuit` in the documentation for all available operations."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "77878c23",
+   "metadata": {},
+   "source": [
+    "## Run your quantum program\n",
+    "\n",
+    "Quantum circuits can produce random results, so you'll often want to collect a sample of the outputs by running the circuit many times. The following code samples the circuit using the `Sampler` class. \n",
+    "\n",
+    "This tutorial uses Qiskit's built-in `Sampler`, which can only handle small numbers of qubits. You can replace this with samplers of high-performance simulators from [Qiskit Aer](https://github.com/Qiskit/qiskit-aer#readme), or with samplers that send your circuit over the internet to run on real quantum computers."
```

Comment:
when mentioning the real quantum computer option could you explicitly mention "IBM Qiskit Runtime" and link to their docs?